src/franka_plan_runner.cc

```diff
@@ -750,8 +727,8 @@ franka::JointPositions FrankaPlanRunner::JointPositionCallback(
     // A higher speed threshold may result in the benign libfranka exception:
     //    Motion finished commanded, but the robot is still moving!
     //    ["joint_motion_generator_acceleration_discontinuity"]
-    static const double CONV_ANGLE_THRESHOLD {0.001};  // rad, empirical
-    static const double CONV_SPEED_THRESHOLD {0.009};  // rad/s, L2 norm
+    static const double CONV_ANGLE_THRESHOLD {0.0010};  // rad, empirical
+    static const double CONV_SPEED_THRESHOLD {0.0085};  // rad/s, L2 norm
```

Comment:
Still think these should be defined in a header or parameters somewhere

Reply:
is there a place where similar constants are already stored?